### PR TITLE
Update public community docs for creation, deployment, onboarding

### PR DESCRIPTION
**Pull request by mattmoss**

### CODE REVIEW

File: processes/community/onboarding/README.md [hunk 1]
```diff
@@ -5,127 +5,89 @@ Complete the instructions below, and you will be enrolled shortly.
 
 **This process is only available for pre-approved accounts. To request approval, contact [support@opes.one](mailto:support@opes.one)**
 
-**Note**: *These instructions are deprecated and will expire on September 1st, 2019. A new enrollment process will be implemented at that time.*
+**Note**: *These instructions are deprecated and will expire on October 1st, 2019. A new enrollment process will be implemented at that time.*
```
Comment (ladybug) [inline processes/community/onboarding/README.md]: Does this mean they are already expired?

Comment (mattmoss) [reply processes/community/onboarding/README.md]: Yes, but maybe not.

@RocketDapps This is the public-facing copy of the same file from `o1.documentation`. Please comment on whether the files are out of date, or whether we should extend their expiration.

Comment (mattmoss) [reply processes/community/onboarding/README.md]: @ladybug I believe Brendan is on vacay until Tues, so make sure you follow up with him after tomorrow.  If you don't hear from him, ping me.

Comment (ladybug) [reply processes/community/onboarding/README.md]: October 1st, 2019 is in the past?

**Comment by mattmoss** [comment]
Part of the issue is that getting this doc from internal to public-facing has taken a while...  lower priority on my list.  Which is one of the reasons to update the process.

That being said, that's why I want to check with @RocketDapps on this.  If it's simply updating the date, then we can do that.  If we really expect this to be outdated, I'm fine updating the site (since it's more up-to-date than what's currently public)....

Really, this speaks more to process than content, since I expect the docs to be fine. But I want to check all this with @RocketDapps when he's back.

**Pull request merged**